features/step_definitions/basic_steps.rb

```diff
@@ -1,7 +1,41 @@
 When(/^I visit "([^"]*)" page$/) do |page_name|
-  visit root_path
+
+  page_path = get_path page_name
```

Comment:
I can see the need for clarity here in defining the extra variable. But let's say your method `get_path` was named `page_path_from`, then you could have a one liner that says

```ruby
visit page_path_from page_name
```

_Could probably have a better name than that_ 😉 

But you can see how that reads a lot better, and there's really no need for the extra variable in this case. Because If I look around I can see that there's a method that takes a `page_name` as an argument and returns a path that I can visit using the `visit` method